admin/app/components/solidus_admin/orders/index/component.rb

```diff
@@ -33,7 +33,13 @@ def batch_actions
   end
 
   def filters
-    []
+    [
```

Comment:
Nit: there's a typo in the corresponding commit message: "between the old and new ~~product~~ order index page".